### DISCUSSION TITLE
nextcloud-aio-mastercontainer continually restarts (different log than I have seen before)

### DISCUSSION THREAD

**Discussion by fleshcrucifix** [nextcloud/all-in-one#3221] [Questions] [answered]
My nextcloud-aio-mastercontainer continually restarts as described by others.  I am hoping mine is an easier fix (I have searched through similar discussions)... the log says the same thing over and over again: Docker run command for AIO is incorrect as a CMD option was given which is not expected.

I am not a total CLI noob, but I am having trouble modifying files in the docker container - I can find them but don't know how to change them.

**Comment by szaimen** [comment, marked as answer]
> modifying the docker container

You need to first stop it, remove it and start it again with the modified command.
